src/Palette.elm

```diff
@@ -0,0 +1,28 @@
+module Palette exposing (..)
```

Comment:
I'd think of different name if the module contains something else than just colours. It wouldn't be my pick when I'd look for font sizes, spacings etc..

Reply:
The reason for this module name is actually something I got from Matt Griffith's talk on elm-ui [at 23:35](https://youtu.be/Ie-gqwSHQr0?t=1414), where he basically mentions that it's a good idea to design starting with a palette, which includes colors, along with fonts used, sizing, etc.

I understand this can be easily mistaken to only stand for Palette of colors, so perhaps a more descriptive name would be `UIPalette`.